README.md

```diff
@@ -1,4 +1,28 @@
-Disclaimer: suboptimal in many ways, but more than enough for what I need.
+# Branch search
+
+A simple terminal branch search for git
+
+## Install
+
+You can install branchsearch by cloning the repo:
+
+`git clone https://github.com/npazosmendez/branchsearch.git`
+
+Go into the repository's folder and running (if you want it installed automatically):
```

Comment:
This sentence looks weird :thinking:

Reply:
Indeed. I think I rewrote part of it so that's probably why.